fwutil/main.py

```diff
@@ -12,6 +12,7 @@
     from .lib import PlatformDataProvider, ComponentStatusProvider, ComponentUpdateProvider
     from .lib import URL, SquashFs
     from .log import LogHelper
+    from sonic_py_common import device_info
```

Comment:
To simplify dependencies management imports should be grouped in next order:
```
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Interesting from https://www.python.org/dev/peps/pep-0008/#imports

```
# Correct:
import os
import sys
# Wrong:
import sys, os
```
Is it affect logic of work or its just a recommendation for style of code?

Reply:
done